lib/generators/apicasso/install/install_generator.rb

```diff
@@ -16,6 +18,7 @@ def self.next_migration_number(path)
         @prev_migration_nr.to_s
       end
 
+      # Move Apicasso tables migration to correct directory
```

Comment:
Create a migration to setup database tables used by the engine to implement authentication, authorization and auditability.